src/Link.ts

```diff
@@ -40,7 +56,7 @@ export class Link extends WidgetBase<LinkProperties> {
 
 	protected render(): DNode {
 		const { to } = this.properties;
-		const props = { ...this.properties, to: undefined, isOutlet: undefined, params: undefined, routerKey: undefined };
+		const props = { ...this.properties, onclick: this._onClick, onClick: undefined, to: undefined, isOutlet: undefined, params: undefined, routerKey: undefined, router: undefined };
```

Comment:
Open to suggestions on a better way of "removing" properties - can also do something like 

```ts
const { to, isOutlet, ...props } = this.properties;
```

But then of course you get the unused variable linting errors.